pkg/image/validation/validation.go

```diff
@@ -0,0 +1,27 @@
+package validation
+
+import (
+	"github.com/suse-edge/edge-image-builder/pkg/image"
+)
+
+type FailedValidation struct {
```

Comment:
Nit: `ValidationFailure` sounds much better in what the structure is conveying in my opinion.

Reply:
I went back and forth. I liked this one since in my head I kept thinking of the list of them as "a list of failed validations" but I could just as easily wrap my head around "a list of validation failures".